packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md

```diff
@@ -21,20 +21,20 @@ This page provides the API rate limits for authentication and end-user activitie
 | Action and Okta API Endpoint                                                                                           | Developer (free) | Developer (paid) | One App | Enterprise | Workforce Identity    |
 | ---------------------------------------------------------------------------------------------------------------------- | ----------------: | ----------------: | -------: | ----------: | ---------------------: |
 | **Cumulative rate limit**                                                                                              | **1,600**        | **3,800**        | **4,200**| **10,650**| **13,300**            |
```

Comment:
Please update the "Cumulative rate limit" section for One App and Dev Paid to match the increase we are doing at the endpoint level.  Pls double check but I think Dev Paid and One App should be at 8,400. @simranbrar-okta, after we decide what to do with /wsfed endpoints for One App, we can talk about combining One App and Enterprise and calling it "Customer Identity".  We should also get Yuliya's feedback on that.

Reply:
@simranbrar-okta - could you please provide the Cumulative rate limit changes? Thank you.

Reply:
Correct.  The main hold up is with `/wsfed` that needs analysis before we can decide if we can include it in OneApp. @simranbrar-okta - FYI

Reply:
@haozhao-okta can you pls provide the cumulative limits based on your PR?

Reply:
Confirmed the cumulative limits for one-app and dev paid are 8400 @simranbrar-okta  - FYI

Reply:
for ALL of them, or just for Auth/end user? What are the values for management and other endpoint categories